src/Config/permissions.rs

```diff
@@ -106,17 +106,11 @@ impl BridgePermissions {
         service: String,
         permission: String,
     ) -> napi::Result<bool> {
-        let parts: Vec<&str> = mxid.split(':').collect();
-        let domain: String;
+        let server_name = mxid.split_once(":").unwrap().1.to_owned();
```

Comment:
Reivilibre came up with this one. :pray: 
Poljar and reivilibre recommended to use Ruma instead of re-writing a parser.

https://docs.rs/ruma-identifiers/latest/ruma_identifiers/struct.UserId.html